solr/core/src/java/org/apache/solr/response/PrometheusResponseWriter.java

```diff
@@ -49,6 +49,7 @@
  */
 @SuppressWarnings(value = "unchecked")
 public class PrometheusResponseWriter extends RawResponseWriter {
+  static String CONTENT_TYPE_PROMETHEUS = "text/plain; version=0.0.4";
```

Comment:
obviously should be private and final.

IMO is also a pointless constant but we have different preferences on this